docker/docker-compose.yml

```diff
@@ -173,6 +173,35 @@ services:
       configuration:
         condition: service_started
 
+  cache2:
```

Comment:
can you rename this to query-cache?

Can you also add a profile for this, so that it won't be run by default?  We don't want to run this way by default because the github actions which run these services have limited resources.

something like this would work:
```
query-cache:
    profiles:
        - querycache
```